libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -214,6 +215,6 @@ export const reviewCreateTab = (
         }
       }
     },
-    secondaryButtonDisabled: isCreating,
+    secondaryButtonDisabled: isCreating || (!previousTabId && isTemplateNameLocked),
```

Comment:
Why do we have this logic in all secondary buttons related to template locked? in reviewCreateTab previousTabId is always true right?